src/server/repository/adapter/col.ts

```diff
@@ -9,11 +9,12 @@ export interface ColDB {
   row_id: number
 }
 
-export const ColAdapter = (
-  { props: { labels, style, variableNo, calculateFn, validateFns, ...rest }, ...col }: ColDB,
-  _?: unknown,
-  __?: unknown
-): Col => {
+export const ColAdapter = (colDB: ColDB): Col => {
+  const {
+    props: { labels, style, variableNo, calculateFn, validateFns, ...rest },
+    ...col
+  } = colDB
+
```

Comment:
_ and __ was added to support db.map and arr.map, does it work without?

Reply:
this change doesn't give any problem, from what I saw